### PR TITLE
Room FAQs: Display raw text in /rfaq

### PR DESCRIPTION
**Pull request by Distrib-ps**
Pr for https://www.smogon.com/forums/threads/add-editdaily-and-editfaq-commands.3676502/

### CODE REVIEW

**Review by AnnikaCodes — DISMISSED** [review]
nvm

File: server/chat-plugins/room-faqs.ts [hunk 1]
```diff
@@ -120,6 +120,12 @@ export const commands: ChatCommands = {
 
 		if (!this.runBroadcast()) return;
 		this.sendReplyBox(Chat.formatText(roomFaqs[room.roomid][topic], true));
+		if (!this.broadcasting && user.can('ban', null, room, 'rfaq')) {
+			const arg = roomFaqs[room.roomid][topic];
+			this.sendReplyBox(
+				Utils.html`<code>/addfaq ${topic}, ${arg} </code>`
+			);
```
Comment (AnnikaCodes) [inline server/chat-plugins/room-faqs.ts:127]:
```suggestion
			Chat.getReadmoreCodeBlock(Utils.html`/addfaq ${topic}, ${arg}`);
```
Actually, no. This should use a readmore block, I'm fairly sure.

Comment (Distrib-ps) [reply server/chat-plugins/room-faqs.ts:127]: If i do that, he break link http://prntscr.com/xa782h

File: server/chat-plugins/room-faqs.ts [hunk 2]
```diff
@@ -120,6 +120,10 @@ export const commands: ChatCommands = {
 
 		if (!this.runBroadcast()) return;
 		this.sendReplyBox(Chat.formatText(roomFaqs[room.roomid][topic], true));
+		if (!this.broadcasting && user.can('ban', null, room, 'rfaq')) {
+			const arg = roomFaqs[room.roomid][topic];
+			this.sendReplyBox(Chat.getReadmoreCodeBlock(`/addfaq ${topic}, ${arg}`));
+		}
```
Comment (Zarel) [inline server/chat-plugins/room-faqs.ts:126]: I think sources should be entirely hidden by default.

```suggestion
		if (!this.broadcasting && user.can('ban', null, room, 'rfaq')) {
			const code = Utils.escapeHTML(roomFaqs[room.roomid][topic]).replace(/\n/g, '<br />');
			this.sendReplyBox(`<details><summary>Source</summary><code style="white-space: pre-wrap; display: table; tab-size: 3">/addfaq ${topic}, ${arg}</code></details>`);
		}
```

This matches `/roomintro` better, too.

Comment (Distrib-ps) [reply server/chat-plugins/room-faqs.ts]: Indeed, it should also be done for `/roomevent view `?

Comment (Zarel) [reply server/chat-plugins/room-faqs.ts]: Possibly. Do you want to do that in this PR, too?

Comment (Distrib-ps) [reply server/chat-plugins/room-faqs.ts]: Yes i will do that

Comment (Distrib-ps) [reply server/chat-plugins/room-faqs.ts]: I can't seem to do it in this PR. Can I do another one?

**Comment by Zarel** [comment]
Thanks!

I'm very concerned by what you mean by "I can't seem to do it in this PR". How is that even possible? What did you try to click or type, and what happened when you did it?

**Comment by Distrib-ps** [comment]
> Thanks!
> 
> I'm very concerned by what you mean by "I can't seem to do it in this PR". How is that even possible? What did you try to click or type, and what happened when you did it?

I can’t find how PR another file, in this PR.

**Comment by Zarel** [comment]
Click this:

**Comment by Zarel** [comment]
But also: You should just learn to use Git.

**Comment by Distrib-ps** [comment]
I see, I am new to Git. I will learn how to use it. Thank you!

**Pull request merged**